### ISSUE TITLE
Research Request - Accessibility for GTFS Digest Products

### ISSUE THREAD

**Issue by amandaha8** [cal-itp/data-analyses#1711]
## Research Question
There are new accessibility warning messages that pop up when we are about to deploy our portfolio. These warnings should be addressed. 

<img width="895" height="299" alt="Image" src="https://github.com/user-attachments/assets/964cce73-2faa-4eea-91d3-3e97d652aa71" />

## How will this research be used?
Make our portfolio accessible to all users. 

## Stakeholders & End-Users
Users of our website and the State of California's policy to make our webpages accessible. 

## Timeline of deliverables
TBD

## Products

- [ ] Operator grain
- [ ] Legislative District
- [ ] Caltrans Distirct

## Altair
- [ ] Add zoom to Altair charts so people can see the figures better.
- [ ] Widen Altair charts so they are larger and easier to see.
- [ ] Change font size.

**Comment by jparr** [comment]
A note on this: I ran the axe-check on many of the portfolios through the new calitp-portfolio cli tool https://github.com/cal-itp/data-analyses/pull/2059

Most of these odd warnings went away because the new tool does a better job of rendering the pages with full context. The only errors that I saw on most pages were Altair charts. I guess that leaves this ticket open as potential research topic to see if they can be improved but overall the health of these sites is much better than the previous assessment reflected.
